x11-misc/dsx/dsx-0.1-r1.ebuild

```diff
@@ -0,0 +1,30 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+PYTHON_COMPAT=( python2_7 )
+inherit python-r1
```

Comment:
use `python-single-r1` (if you have just one impl in `PYTHON_COMPAT`, you want to avoid unnecessary wrapping)